core/model/src/commonMain/kotlin/org/mifos/mobile/core/model/entity/TransactionDetails.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2025 Mifos Initiative
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * See https://github.com/openMF/mobile-mobile/blob/master/LICENSE.md
+ */
+package org.mifos.mobile.core.model.entity
+
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonNames
+import org.mifos.mobile.core.model.entity.client.Type
+
+@Serializable
+data class TransactionDetails(
+
+    val id: Long? = null,
+
+    val officeId: Long? = null,
+
+    val officeName: String? = null,
+
+    @OptIn(ExperimentalSerializationApi::class)
+    @JsonNames("transactionType")
```

Comment:
You could use this instead:

```suggestion
    @SerialName("transactionType")
```

Ref: [SerialName Official documentation](https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-core/kotlinx.serialization/-serial-name/)